src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -1682,10 +1682,10 @@ bool esbmc_parseoptionst::process_goto_program(
       goto_k_induction(goto_functions);
     }
 
-    if(cmdline.isset("goto-contractor"))
+    if(cmdline.isset("goto-contractor") || cmdline.isset("goto-contractor-algo2"))
```

Comment:
We should find a better name for "goto-contractor-algo2". What is "algo2"?

Reply:
changed to goto-contractor-condition.